### DISCUSSION TITLE
走完流程还是不显示降雨通知

### DISCUSSION THREAD

**Discussion by CccZzzBbbb** [NSRingo/.github#21] [Q&A 问答] [answered]
![image](https://github.com/user-attachments/assets/45eac7b9-2439-4606-a92c-95d0c1de7437)![image](https://github.com/user-attachments/assets/8a2bc4e6-2b4f-4572-b991-21ebfbe345b7)
用surge添加了插件还是没有降雨通知
求帮助

**Comment by cat-kun** [comment, marked as answer]
https://github.com/orgs/NSRingo/discussions/1#discussioncomment-10928722
